lib/PuppeteerSharp/Launcher.cs

```diff
@@ -82,14 +82,13 @@ public Launcher(ILoggerFactory loggerFactory = null)
         /// The method launches a browser instance with given arguments. The browser will be closed when the Browser is disposed.
         /// </summary>
         /// <param name="options">Options for launching Chrome</param>
-        /// <param name="chromiumRevision">The revision of Chrome to launch.</param>
         /// <returns>A connected browser.</returns>
         /// <remarks>
         /// See <a href="https://www.howtogeek.com/202825/what%E2%80%99s-the-difference-between-chromium-and-chrome/">this article</a>
         /// for a description of the differences between Chromium and Chrome.
         /// <a href="https://chromium.googlesource.com/chromium/src/+/lkcr/docs/chromium_browser_vs_google_chrome.md">This article</a> describes some differences for Linux users.
         /// </remarks>
-        public async Task<Browser> LaunchAsync(LaunchOptions options, int chromiumRevision)
+        public async Task<Browser> LaunchAsync(LaunchOptions options)
```

Comment:
I'm thinking about leaving this method with the `chromiumRevision` and mark it as Obsolete.
What do you think @Meir017 ?

Reply:
because it's a breaking change?

Reply:
Yes, this will break user's code if they are using the `Downloader` class, but maybe we can make their life easier?

Reply:
I can buy this